include/seqan3/io/sam_file/output.hpp

```diff
@@ -592,6 +613,9 @@ class sam_file_output
 
 protected:
     //!\privatesection
+    //!\brief Whether any record has been written. Needed on deconstruction to know if a header still must be written.
```

Comment:
This reads strangely to me, maybe
```suggestion
    //!\brief Whether any record has already been written is needed during deconstruction to know whether a header still needs to be written.
```